index.js

```diff
@@ -1,5 +1,16 @@
 'use strict';
 
+exports.getQueryString = function (maybeUrl) {
```

Comment:
I too, but you have to read it as it's going to be used `queryString.extract()`. `queryString.getQueryString()` feels very verbose.